Dockerfile

```diff
@@ -1 +1,7 @@
-FROM golang:1.4.1-onbuild
+FROM golang:1.6-alpine
+
+MAINTAINER Tom Denham <tom@tigera.io>
+
+RUN apk -U add bash git musl-dev gcc linux-headers
+WORKDIR /go/src/github.com/coreos/flannel
+
```

Comment:
@tomdee In that link they are disabling cgo support for the build, but flannel needs cgo.

I checked wheezy, and it does not have aarch64 compiler support.

I think an answer is to use a recent debian or ubuntu image and install the cross compiler packages (gcc-aarch64-linux-gnu and libc6-dev-arm64-cross), then use a gimme script that can install a go compiler with arm64 cgo.  See https://github.com/coreos/ignition/pull/235/commits/787b5104676d2477bd460eac343c1d307d7dc5a5